### ISSUE TITLE
Rendering error for nested table for `\begin{tabular}...\end{tabular}`

### ISSUE THREAD

**Issue by OlgaRedozubova** [Mathpix/mathpix-markdown-it#174]
LaTex:
```
\begin{tabular} { l }  \( \qquad \)  \begin{tabular} { | l | l | l | l | l | l | l | l | l | } \hline \end{tabular} \\ \hline 4 \end{tabular}
```
<img width="236" alt="Screen Shot 2021-12-01 at 11 25 54" src="https://user-images.githubusercontent.com/32493105/144207749-b8062a05-7b67-41ce-a978-b0cb49455f6c.png">

This occurs when a formula and a nested table are in the same cell.

If to split it into two cells, it will work:
<img width="1003" alt="Screen Shot 2021-12-01 at 11 29 04" src="https://user-images.githubusercontent.com/32493105/144208541-392d1af4-8fac-4093-8142-deb4c45f484c.png">

**Comment by dlrac** [comment]
Another example:
```
\begin{tabular}{|c|c|c|c|}
\hline \multirow{2}{*}{\( m^{\prime}, n^{\prime}, \lambda^{\prime} \)} & \multicolumn{3}{|l|}{\( m, n, \lambda \)} \\
\hline & \( 3,1,1 \) & 2,2,1 & \( 2,1,0 \) \\
\hline \( 3,1,1 \) & \begin{tabular}{l}
\begin{tabular}{l}
\( R=\left(\begin{array}{ccc}\mu & \nu & \sigma \\
\nu & \alpha & 1 \\
\sigma & 1 & 0\end{array}\right) \) \\
\( E=\alpha-2 \sigma \)
\end{tabular}
\end{tabular} & \begin{tabular}{l}
\begin{tabular}{c}
\( R=\left(\begin{array}{ccc}\nu & \alpha & 0 \\
\mu & \sigma & \omega\end{array}\right) \) \\
\( E=\left(\begin{array}{cc}\kappa & \kappa \\
\omega-\alpha\end{array}\right) \)
\end{tabular}
\end{tabular} & \begin{tabular}{l}
\begin{tabular}{c}
\( R=\left(\begin{array}{ccc}\nu & \alpha & \omega \\
\mu & \sigma & 0\end{array}\right) \) \\
\( E=\kappa \)
\end{tabular}
\end{tabular} \\
\hline 2,2,1 & \begin{tabular}{l}
\( R=\left(\begin{array}{ll}\mu & \nu \\
\sigma & \alpha \\
\omega & 0\end{array}\right) \) \\
\( E=(\kappa(\omega-\alpha)) \)
\end{tabular} & \begin{tabular}{l}
\begin{tabular}{l}
\( R=\left(\begin{array}{ll}\mu & \nu \\
\nu & \kappa\end{array}\right) \) \\
\( E=\left(\begin{array}{ll}\sigma & \omega \\
\omega & \kappa\end{array}\right) \)
\end{tabular}
\end{tabular} & \begin{tabular}{l}
\begin{tabular}{l}
\( R=\left(\begin{array}{ll}\mu & \sigma \\
\nu & \omega\end{array}\right) \) \\
\( E=(\kappa \omega) \)
\end{tabular}
\end{tabular} \\
\hline \( 2,1,0 \) & \begin{tabular}{l}
\begin{tabular}{l}
\( R=\left(\begin{array}{ll}\mu & \nu \\
\sigma & \alpha \\
0 & \omega\end{array}\right) \) \\
\( E=\kappa \)
\end{tabular}
\end{tabular} & \begin{tabular}{l}
\( R=\left(\begin{array}{ll}\mu & \nu \\
\sigma & \omega\end{array}\right), E=\binom{\kappa}{\omega} \)
\end{tabular} & \begin{tabular}{l}
\begin{tabular}{l}
\( R=\left(\begin{array}{ll}\mu & \nu \\
\nu & \kappa\end{array}\right) \) \\
\( E=\sigma \)
\end{tabular}
\end{tabular} \\
\hline 2,1,1 & \begin{tabular}{l}
\begin{tabular}{l}
\( R=\left(\begin{array}{cc}\mu & \nu \\
\sigma & \alpha \\
\omega & 0\end{array}\right) \) \\
\( E=\omega-\alpha \)
\end{tabular}
\end{tabular} & \begin{tabular}{l}
\( R=\left(\begin{array}{cc}\mu & \nu \\
0 & \omega\end{array}\right), E=\binom{\alpha}{\sigma} \)
\end{tabular} & \begin{tabular}{l}
\begin{tabular}{l}
\( R=\left(\begin{array}{cc}\mu & \sigma \\
0 & \nu\end{array}\right) \) \\
\( E=\kappa \)
\end{tabular}
\end{tabular} \\
\hline 2,0,0 & \begin{tabular}{l}
\( R=\left(\begin{array}{ll}\mu & \nu \\
\sigma & \alpha \\
\alpha & 0\end{array}\right) \) \\
\( E \) not existing
\end{tabular} & \begin{tabular}{l}
\( R=\left(\begin{array}{ll}\mu & \nu \\
\omega & 0\end{array}\right) \) \\
\( E \) not existing
\end{tabular} & \begin{tabular}{l}
\( R=\left(\begin{array}{ll}\mu & \nu \\
\sigma & 0\end{array}\right) \) \\
\( E \) not \\
existing
\end{tabular} \\
\hline \( 1,2,1 \) & \begin{tabular}{l}
\begin{tabular}{l}
\( R=\left(\begin{array}{c}\mu \\
\nu \\
\alpha\end{array}\right) \) \\
\( E=(\omega \alpha) \)
\end{tabular}
\end{tabular} & \begin{tabular}{l}
\( R=\binom{\kappa}{\sigma}, E=\left(\begin{array}{ll}\mu & \nu \\
\omega & 0\end{array}\right) \)
\end{tabular} & \begin{tabular}{l}
\begin{tabular}{l}
\( R=\binom{\mu}{\nu} \) \\
\( E=\left(\begin{array}{ll}\sigma & 0\end{array}\right) \)
\end{tabular}
\end{tabular} \\
\hline \( 1,1,0 \) & \begin{tabular}{l}
\( R=\left(\begin{array}{l}\mu \\
\nu \\
\alpha\end{array}\right), E=\alpha \)
\end{tabular} & \begin{tabular}{l}
\( R=\binom{\kappa}{\sigma}, E=\binom{\mu}{0} \)
\end{tabular} & \begin{tabular}{l}
\begin{tabular}{l}
\( R=\binom{\mu}{\nu} \) \\
\( E=\sigma \)
\end{tabular}
\end{tabular} \\
\hline \( 1,1,1 \) & \begin{tabular}{l}
\( R=\left(\begin{array}{l}0 \\
\nu \\
\alpha\end{array}\right), E=\omega \)
\end{tabular} & \begin{tabular}{l}
\( R=\binom{\kappa}{\sigma}, E=\binom{\mu}{\nu} \)
\end{tabular} & \begin{tabular}{l}
\begin{tabular}{l}
\( R=\binom{\mu}{\nu} \) \\
\( E=0 \)
\end{tabular}
\end{tabular} \\
\hline \( 1,0,0 \) & \begin{tabular}{l}
\( R=\left(\begin{array}{c}\mu \\
\alpha \\
0\end{array}\right) \) \\
\( E \) not existing
\end{tabular} & \begin{tabular}{l}
\( R=\binom{\kappa}{\sigma} \) \\
\( E \) not existing
\end{tabular} & \begin{tabular}{l}
\( R=\binom{\kappa}{\sigma} \) \\
\( E \) not \\
existing
\end{tabular} \\
\hline \( 0,1,0 \) & \begin{tabular}{l}
\( R \) not existing, \\
\( E=\alpha \)
\end{tabular} & \begin{tabular}{l}
\begin{tabular}{l}
\hline\( R \) not \\
existing, \\
\( E=\binom{\kappa}{\sigma} \)
\end{tabular}
\end{tabular} & \begin{tabular}{l}
\( E=\alpha, R \) not \\
existing
\end{tabular} \\
\hline
\end{tabular}
```

**Comment by OlgaRedozubova** [comment]
Fixed in [mathpix-markdown-it@2.0.3](https://github.com/Mathpix/mathpix-markdown-it/pull/348)
